release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -122,6 +122,26 @@ Release Team selection should happen in accordance with the [Release Team select
   - Kubernetes 1.16: Unlimited Breadsticks For All
   - Kubernetes 1.17: The Chillest Release - Capynetes
 
+### Mailing List Standards
+
+Important communications and discussions should be posted in the release team
+mailing list https://groups.google.com/forum/#!forum/kubernetes-release-team.
+
+This list is owned by SIG Release and as such the SIG Release chairs are owners
+of it.
+While SIG Release leads are owners of this list, the release team lead and
+release lead shadows will serve as managers of the list.
+
+The release team lead and release lead shadows should periodically go through
+the release team mailing list and prune members from the last release who are
+no longer part of the release team and prune inactive members.
+
+The membership criteria in general is that any member of the release team,
```

Comment:
Membership of criteria is:
- Current Release Team, including Emeritus Adviser(s)
- SIG Chairs
- ALL Branch Managers

The Patch Release Team doesn't need to be on.